plugins/woocommerce/tests/php/src/Internal/Admin/ReviewsListTableTest.php

```diff
@@ -11,4 +12,23 @@
  */
 class ReviewsListTableTest extends WC_Unit_Test_Case {
 
+	/**
+	 * @covers \Automattic\WooCommerce\Internal\Admin\ReviewsListTable::get_columns()
+	 * @TODO Change the `screen` to be our page when it gets created in MWC-5330
+	 */
+	public function test_get_columns() {
+		$this->assertSame(
+			[
+				'cb'       => '<input type="checkbox" />',
+				'type'     => _x( 'Type', 'review type', 'woocommerce' ),
+				'author'   => __( 'Author', 'woocommerce' ),
+				'rating'   => __( 'Rating', 'woocommerce' ),
+				'comment'  => _x( 'Review', 'column name', 'woocommerce' ),
+				'response' => __( 'Product', 'woocommerce' ),
+				'date'     => _x( 'Submitted on', 'column name', 'woocommerce' ),
+			],
+			( new ReviewsListTable( [ 'screen' => 'comments' ] ) )->get_columns()
```

Comment:
The test was failing if I didn't pass in a `screen` parameter because of stuff the base list table class does in the constructor. I've set it to `comments` for now, which doesn't impact the test at all, but when we create our custom page we can change it to that.